apis/v1alpha1/generator.yaml

```diff
@@ -663,6 +671,11 @@ resources:
         from:
           operation: DescribeNotebookInstance
           path: Url
+      stoppedByControllerMETA:
+        is_read_only: true
+        from:
+          operation: DescribeNotebookInstance
+          path: Url
```

Comment:
this is weird 
can we not have the value as null instead ?

Reply:
Not really since code generator needs the shape(string/map/array) the generate the status.